lib/stack_master/commands/drift.rb

```diff
@@ -0,0 +1,115 @@
+require 'diffy'
+
+module StackMaster
+  module Commands
+    class Drift
+      include Command
+      include Commander::UI
+
+      DETECTION_COMPLETE_STATES = [
+        'DETECTION_COMPLETE',
+        'DETECTION_FAILED'
+      ]
+
+      def perform
+        detect_stack_drift_result = cf.detect_stack_drift(stack_name: stack_name)
+        drift_results = wait_for_drift_results(detect_stack_drift_result.stack_drift_detection_id)
+
+        puts colorize("Drift Status: #{drift_results.stack_drift_status}", stack_drift_status_color(drift_results.stack_drift_status))
+        return if drift_results.stack_drift_status == 'IN_SYNC'
+
+        failed
+
+        resp = cf.describe_stack_resource_drifts(stack_name: stack_name)
+        resp.stack_resource_drifts.each do |drift|
+          display_drift(drift)
+        end
+      end
+
+      private
+
+      def cf
+        @cf ||= StackMaster.cloud_formation_driver
+      end
+
+      def display_drift(drift)
+        color = drift_color(drift)
+        puts colorize("#{drift.stack_resource_drift_status} #{drift.resource_type} #{drift.logical_resource_id} #{drift.physical_resource_id}", color)
```

Comment:
I rarely care about line length but it's making it a little hard to review this PR.